samples/golang/webfrontend/src/app/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+	"html/template"
+	"log"
+	"net/http"
+	"strconv"
+	"sync"
+)
+
+// ServerMessage is a representation of a message from server
+type ServerMessage struct {
+	Value string
+}
+
+// PageVariables is a representation of a server page
+type PageVariables struct {
+	PageTitle   string
+	PageMessage ServerMessage
+	PageButton  string
+}
+
+var counter int
+var mutex = &sync.Mutex{}
+
+func main() {
+	http.HandleFunc("/", HomePage)
+	http.HandleFunc("/api", func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintf(w, "Hello from webfrontend")
+	})
+	log.Fatal(http.ListenAndServe(":80", nil))
+}
+
+// HomePage is triggered on load of the server
+func HomePage(w http.ResponseWriter, r *http.Request) {
+	Title := "Server"
```

Comment:
Is it normal to name local variables using Pascal casing?

Reply:
I guess not. I will change it to Camel casing.